src/leases/components/leaseSections/contract/Contract.js

```diff
@@ -27,40 +27,75 @@ const Contract = ({contract}: Props) => {
             <Column medium={4}>
               <label>Sopimusnumero</label>
               <p>{contract.contract_number ? get(contract, 'contract_number', '–') : '–'}</p>
-
-              <label>Järjestelypäätös</label>
-              <p>{contract.setup_decision ? get(contract, 'setup_decision', '–') : '–'}</p>
-
-              <div className='multiple-textrows'>
-                <label>Panttikirjat (numero ja päivämäärä)</label>
-                {contract.pledge_books.map((pledge_book) =>
-                  <p className='text-no-margin' key={get(pledge_book, 'pledge_book_number', '')}>{get(pledge_book, 'pledge_book_number', '')}, {helpers.formatDate(get(pledge_book, 'pledge_book_date', ''))}</p>
-                )}
-              </div>
             </Column>
             <Column medium={4}>
               <label>Allekirjoituspäivämäärä</label>
               <p>{contract.signing_date ? helpers.formatDate(get(contract, 'signing_date', '–')) : '–'}</p>
-
-              <label>Vuokravakuusnumero</label>
-              <p>{contract.lease_deposit_number ? get(contract, 'lease_deposit_number', '–') : '–'}</p>
-
+            </Column>
+            <Column medium={4}>
+              <label>Kommentti allekirjoitukselle</label>
+              <p>{contract.signing_date_comment ? get(contract, 'signing_date_comment', '–') : '–'}</p>
```

Comment:
Default string (-) is defined twice. You get same end result with
<p>{get(contract, 'signing_date_comment', '–')}</p>

Same thing on rows 43, 59, 73, 87, 91, 95

Reply:
The purpose is to show '–' also if null